### ISSUE TITLE
Given code for the challenge does not show

### ISSUE THREAD

**Issue by kalunlee136** [freeCodeCamp/freeCodeCamp#13704]
Challenge [interpolate-a-string-using-backquotes](https://beta.freecodecamp.com/en/challenges/es6/interpolate-a-string-using-backquotes) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/56.0.2924.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

When I go to this challenge, the code that usually shows in the editor is not there. The editor is also stuck constantly refreshing. In addition, you can't write anything in the editor either. I have tried opening in incognito chrome, firefox, and IE11 and the problem is the same.

I checked Chrome dev tools console and it throws a 'Uncaught Error: Minified React error #32'

```js


const result = {
    success: ['max_length', 'no-amd', 'prefer-arrow-functions'],
    failure: ['no-var', 'var-on-top', 'linebreak'],
    skipped: ['id-blacklist', 'no-dup-keys']
}
/* Alter code below this line */
const resultDisplay = undefined;
/* Alter code above this line */
console.log(resultDisplay);
/**
 * 
 * should look like this
 * <li class="text-warning">no-var</li>
 *  <li class="text-warning">var-on-top</li>
 *  <li class="text-warning">linebreak</li>
 **/

```

**Comment by dna113p** [comment]
I see the same thing!

This was my first real look at the code base, and I don't know what the next place to look would be, but when I removed all text from the seed description for this challenge and re seed it, the challenge will load. I tried to pinpoint exactly the cause but I couldn't find it.

**Comment by Marmiz** [comment]
To whom it may concern:

error thrown:
> Unable to find element with ID 50.

**Comment by systimotic** [comment]
Thanks a lot for reporting @kalunlee136, and for investigating @dna113p and @Marmiz!
It looks like @szib is already working on a fix in #13712 👏

**Comment by dhcodes** [comment]
Closed per conversation in #13712
